test/python/test_slice.py

```diff
@@ -61,26 +60,26 @@ def test_strided_slice(self):
     inp = np.random.rand(4, 5).astype("f")
     slice_ts = []
     expected = []
-
     a = np.array([float(x) for x in inp.ravel(order="C")])
     a.shape = (4, 5)
 
     x = tf.placeholder(dtype=dtypes.float32)
+
     slice_ts.append(x[:])
     slice_ts.append(x[...])
     slice_ts.append(x[:, :])
     slice_ts.append(x[:, ...])
     slice_ts.append(x[1:, :-2])
     slice_ts.append(x[::2, :-2])
     slice_ts.append(x[1, :])
-    #slice_ts.append(x[:, 1])
+    slice_ts.append(x[:, 1])
     slice_ts.append(x[1, 1])
     slice_ts.append(x[0])
     slice_ts.append(x[0][1])
     slice_ts.append(x[-1])
```

Comment:
Are these all the slicing cases that we can possibly see. Even if we do not support them now, can we write them down and comment them out as future goals or known unsupported features.
For one, I think this is not supported: y = x[:, tf.newaxis]

Reply:
Yes, there are many such cases that are unsupported. It will actually take me a while to jot down all the unsupported test cases.

We can think of this PR to enable test cases where we are shrinking one or more axis.
slice_ts.append(x[:,1]) or 
for a (2,4,5) shaped tensor
slice_ts.append(x[1,:,:])

Reply:
Added x[:, tf.newaxis] as currently unsupported